app/renderer/lib/client-frameworks/java.js

```diff
@@ -119,65 +105,72 @@ ${this.indent(code, 4)}
     return `${this.getVarName(varName, varIndex)}.sendKeys(${JSON.stringify(text)});`;
   }
 
-  codeFor_back () {
-    return `driver.navigate().back();`;
-  }
-
-  // Change TouchAction to Sequence.
   codeFor_tap (varNameIgnore, varIndexIgnore, pointerActions) {
     const {x, y} = this.getTapCoordinatesFromPointerActions(pointerActions);
-
     return `
-    final PointerInput FINGER = new PointerInput(PointerInput.Kind.TOUCH, "FINGER");
-    Point tapPoint = new Point(${x}, ${y});
-    Sequence tap = new Sequence(FINGER, 1);
-        tap.addAction(FINGER.createPointerMove(Duration.ofMillis(0),
-                PointerInput.Origin.viewport(), tapPoint.x, tapPoint.y));
-        tap.addAction(FINGER.createPointerDown(PointerInput.MouseButton.LEFT.asArg()));
-        tap.addAction(FINGER.createPointerMove(Duration.ofMillis(50),
-                PointerInput.Origin.viewport(), tapPoint.x, tapPoint.y));
-        tap.addAction(FINGER.createPointerUp(PointerInput.MouseButton.LEFT.asArg()));
-        driver.perform(Arrays.asList(tap));
+final PointerInput FINGER = new PointerInput(PointerInput.Kind.TOUCH, "finger");
```

Comment:
java linter usually freaks out if it detects local variables named in uppercase. the rule of thumb is to use camelCase for local variable names. Uppercase is only used for class-level constants

Reply:
Ok, I just read that all caps is recommended for `final` variables, but I'll change it to camelcase